TFlippy_TerritoryControl_Weapons_Dev/Entities/Guns/Flashlight/Flashlight.as

```diff
@@ -29,31 +29,28 @@ void makeLight(CBlob@ this)
 
 void onTick(CBlob@ this)
 {
-	if (isClient())
-	{
-		bool flip = this.isFacingLeft();
-		f32 angle = this.getAngleDegrees();
-
-		Vec2f hitPos;
-		Vec2f dir = Vec2f((flip ? -1 : 1), 0.0f).RotateBy(angle);
-		Vec2f startPos = this.getPosition();
-		Vec2f endPos = startPos + dir * max_distance;
+	bool flip = this.isFacingLeft();
```

Comment:
add isClient back, otherwise flash light eats up a fair bit of server resources (use to be an issue)